### PR TITLE
Company profile API views

### PR DESCRIPTION
**Pull request by monikasulik**
https://uktrade.atlassian.net/browse/ED-146

### CODE REVIEW

**Comment by monikasulik** [comment]
@konradko This is now ready for another look.

File: company/tests/test_views.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+from rest_framework import status
+
+from company.models import Company
+from company.tests import VALID_REQUEST_DATA
+
+
+@pytest.mark.django_db
+def test_company_retrieve_view():
+    client = APIClient()
+    company = Company.objects.create(**VALID_REQUEST_DATA)
```
Comment (konradko) [inline company/tests/test_views.py]: Seeing how often ``VALID_REQUEST_DATA`` is used, maybe we should consider making a pytest 'fixture' out of them. I don't like pytests fixtures though, they are useful, but it's an odd pattern

File: company/views.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+from company.serializers import CompanySerializer
+from company.models import Company
+
+
+class CompanyRetrieveUpdateAPIView(RetrieveUpdateAPIView):
+
+    model = Company
+    serializer_class = CompanySerializer
+
+    def get_queryset(self):
```
Comment (konradko) [inline company/views.py]: is this necessary? Just specifying the model should achieve that I believe

Comment (monikasulik) [inline company/views.py]: I was surprised, but I was getting exceptions when this wasn't here.

File: company/tests/test_views.py (hunk 1 above)
Comment (monikasulik) [inline company/tests/test_views.py]: I kind of copied what was in the codebase already in enrolment. If we're changing it then we need to change both places :)

Comment (konradko) [inline company/tests/test_views.py]: yup, that's what pytest fixtures are for. Fine to leave it, like I mentioned, not a huge fan

File: company/views.py (hunk 2 above)
Comment (konradko) [inline company/views.py]: that is surprising

**Pull request merged**